tests/test_transform.py

```diff
@@ -128,6 +133,29 @@ def test_transforms_pck(self):
 
             self.assertEqual(actual_response, expected_response)
 
+    def test_cora_transforms_pck(self):
+        """Tests the pck transformation for responses that go to the CORA system."""
+
+        test_scenarios = get_cora_test_scenarios("pck")
+
+        print("Found %d cora pck scenarios" % len(test_scenarios))
+
+        for scenario_filename in test_scenarios:
+
+            print("Loading scenario %s " % scenario_filename)
```

Comment:
Should this be going to console or logger?

Reply:
(goes for all lines below)

Reply:
It should be going to the console.  It's meant to just be a helper so that on a failed test, you know what the inputs/outputs are as the way it was written doesn't lend itself to easy debugging.

Would a card in the SDX column to review how the transform-cs tests are done be a fair compromise?  As this repo needs a looking at anyway as there are other issues that make this not as good as it could be (the endpoints for one example).

Reply:
Sure. I'd like to see stuff going to logger rather than console as a general rule :)

Reply:
Created https://trello.com/c/V7tLK87T/690-tidy-up-sdx-transform-cs-endpoints-and-tests to investigate this.  
For tests there shouldn't be a need for messages to be logged to either the console or the logger as the inbuilt diffs you get from assert statements should be enough.  This was kind of an easy way to show that diff